_getting-started/security.md

```diff
@@ -0,0 +1,192 @@
+---
+layout: default
+title: Getting started with OpenSearch security
+nav_order: 60
+---
+
+# Getting started with OpenSearch security
+
+The demo configuration is the most straightforward way to get started with security in OpenSearch. OpenSearch comes bundled with a number of useful scripts, including `install_demo_configuration.sh` (or `install_demo_configuration.bat` for Windows).
+
+This script is located in `plugins/opensearch-security/tools` and performs the following actions:
+
+- Creates demo certificates for TLS encryption on both the transport and REST layers.
+- Configures demo users, roles, role mappings.
+- Configures the Security plugin to use internal database for authentication and authorization.
+- Updates the `opensearch.yml` file with basic configuration needed to get the cluster started.
+
+You can find complete details regarding demo configuration and how to get up and running quickly at [Setting up a demo configuration]({{site.url}}{{site.baseurl}}/security/configuration/demo-configuration/)
+{: .note}
+
+Certain aspects of this configuration, such as demo certificates and default passwords, should never be used in production. These parts of the demo configuration should be updated with your custom information before proceeding to production.
+{: .warning}
+
+## Setting up the demo configuration
+
+Prior to running the `install_demo_configuration.sh` script you must create environment variable named `OPENSEARCH_INITIAL_ADMIN_PASSWORD` set to a strong password. This will be used as the password for the admin user to authenticate with OpenSearch. Use the online tool [_Zxcvbn_](https://lowe.github.io/tryzxcvbn/) to test the strength of any password. After this is set, you can execute `install_demo_configuration.sh` and follow the terminal prompt to enter necessary details.
```

Comment:
```suggestion
Prior to running the `install_demo_configuration.sh` script, you must create an environment variable named `OPENSEARCH_INITIAL_ADMIN_PASSWORD` with a strong password. This will be used as the password for the admin user to authenticate with OpenSearch. Use the online tool [_Zxcvbn_](https://lowe.github.io/tryzxcvbn/) to test the strength of any password. After this, you can execute `install_demo_configuration.sh` and follow the terminal prompt to enter necessary details.
```